python/Scripts/mxcustomnamespacefixer.py

```diff
@@ -0,0 +1,139 @@
+# Fixup code namespaces to be customizable:
+# As run for fixing namespaces in codebase for Github issue #773
```

Comment:
Although this is a useful script to keep around, I wouldn't recommend adding it to MaterialX main, since it's not a script we'd expect other teams to leverage in their own pipelines.